index.html

```diff
@@ -38,6 +34,149 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
+            </div>
+            <div class="btn-card-one">
+                See Project
+            </div>
+        </div>
+        <!-- fourth card start from here -->
+        <div class="card">
+            <div class="card-heading">
+                <img src="images/card-4.svg" alt="card-4">
+            </div>
+            <div class="card-body">
+                <h1 class="card-title">Multi-Post Stories</h1>
+                <div class="card-author">
+                    <h5 class="heading5">CANOPY</h5>
+                    <div class="dot"></div>
+                    <p>Back End Dev</p>
+                    <div class="dot"></div>
+                    <p>2015</p>
+                </div>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+            </div>
+            <div class="card-languages">
+                <p class="language-item ">html</p>
+                <p class="language-item ">css</p>
+                <p class="language-item ">javascript</p>
+            </div>
+            <div class="btn-card-one">See Project </div>
+        </div>
+    </section>
+    <!-- this is section for another card -->
+    <section class="about-section">
+        <h1>About Myself</h1>
+        <p class="about-desc">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
+        <h5 class="btn-connect">LET’S CONNECT</h5>
+        <ul class="social-icon">
+            <li><a href=""><img src="images/icon5.svg" alt="icon-1"></a></li>
+            <li><a href=""><img src="images/icon2.svg" alt="icon-2"></a></li>
+            <li><a href=""><img src="images/icon1.svg" alt="icon-3"></a></li>
+            <li><a href=""><img src="images/icon4.svg" alt="icon-4"></a></li>
+            <li><a href=""><img src="images/icon3.svg" alt="icon-5"></a></li>
+        </ul>
+        <div class="btn-card-one">Get my resume </div>
+        <div class="div-languages">
+            <h4 class="language">Languages</h4>
+                <div class="lang-card">
+                    <img src="images/js.png" alt="js">
+                    javascript
+                </div>
+                <div class="lang-card">
+                    <img src="images/html.png" alt="html">
+                    HTML
+                </div>
+                <div class="lang-card">
+                    <img src="images/css.png" alt="css">
+                    CSS
+                </div>
+                <div class="framworks">
+                    Framworks <span>></span>
+                </div>
+                <hr>
+                <div class="framworks">
+                    Skills <span>></span>
+                </div>
+                <hr>
+        </div>
```

Comment:
- [ ] For lhe technology in each category(e.g., "JavasScript", "HTML" and "CSS") , Please use a list `ul` and each of them a `li` :+1: